### PR TITLE
Extend shutdown allowed time in under min available resources test

### PR DESCRIPTION
**Pull request by brianjohnson5972**
## Change Description
Fixed corner case for failures and added debug logging. And increase time allowed to shutdown.

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: tests/Node.py [hunk 1]
```diff
@@ -1375,10 +1377,13 @@ def verifyAlive(self, silent=False):
             # mark node as killed
             self.pid=None
             self.killed=True
+            if logStatus: Utils.Print("Determined node(formerly pid=%s) is killed" % (pid))
             return False
         except PermissionError as ex:
+            if logStatus: Utils.Print("Determined node(formerly pid=%s) is killed" % (pid))
```
Comment (heifner) [inline tests/Node.py]: This should say "is alive", right?

File: tests/Node.py [hunk 2]
```diff
@@ -1375,10 +1377,13 @@ def verifyAlive(self, silent=False):
...
             return True
 
+        if logStatus: Utils.Print("Determined node(pid=%s) is alive (kill command executed)" % (self.pid))
```
Comment (heifner) [inline tests/Node.py]: This didn't call `kill`, well technically it did, but the kill is kill 0 which is just check the status. I think you should remove "(kill command executed)"
